fluent-plugin-s3.gemspec

```diff
@@ -26,4 +26,5 @@ Gem::Specification.new do |gem|
   # aws-sdk-core requires one of ox, oga, libxml, nokogiri or rexml,
   # and rexml is no longer default gem as of Ruby 3.0.
   gem.add_development_dependency "rexml"
+  gem.add_development_dependency 'zstd-ruby'
```

Comment:
Shouldn't this be `gem.add_dependency`?

Reply:
Right. I also think it should add gem into runtime dependency instead.

Reply:
Thank you for your valuable feedback. I have updated the `fluent-plugin-s3.gemspec` file by moving `zstd-ruby` from development dependencies to runtime dependencies as you suggested

Reply:
@ddukbg 
Sorry, I was wrong.
It should be `development_dependency`.
Please take a look at the following PR for details.

* #445

This fix will require manual installation of `zstd-ruby` gem.